SourceX/discord/discord.cpp

```diff
@@ -0,0 +1,141 @@
+#include "discord.h"
+
+#include <Discord/cpp/discord.h>
+#include <chrono>
+#include <string>
+#include <sstream>
+
+#include "../types.h"
+#include "../Source/player.h"
+
+namespace dvl {
+
+DiscordManager::DiscordManager()
+{
+	discord::Core::Create(DISCORD_DEVIX_APPID, DiscordCreateFlags_NoRequireDiscord, &core);
+}
+
+const char *difficulties[] = { "Normal", "Nightmare", "Hell" };

```

Comment:
`difficulties`, `DifficultyToStr` etc should either be marked `static` or placed in an anonymous namespace

Reply:
Isn't there a global array with tease strings already?

Reply:
They are hardcoded inside of various functions.